src/test/java/org/jqassistant/plugin/jee/cdi/test/set/beans/scope/jakarta/JakartaTypeWithDependentMethod.java

```diff
@@ -0,0 +1,14 @@
+package org.jqassistant.plugin.jee.cdi.test.set.beans.scope.jakarta;
+
+import jakarta.enterprise.context.Dependent;
+
+import org.jqassistant.plugin.jee.cdi.test.set.beans.scope.ProducedBean;
+
+public class JakartaTypeWithDependentMethod {
+
+    @Dependent
```

Comment:
Please verify if this is supported by CDI or if Produces is required in addition. If not then this class and according rules shall be removed.